### ISSUE TITLE
Tutor side - lessons - Calendar view is not showing on mobile view.

### ISSUE THREAD

**Issue by Rajar91221** [Rajar91221/Tutor-reach-QA-#192]
![image](https://user-images.githubusercontent.com/50227023/132303585-148cb32c-5c15-41fc-9055-8c519888d070.png)

**Comment by Adiconquerors** [comment]
@TutorReachAdmin 

We are hiding calendar for mobile view. By default calednar is not viewing in mobile view so we can remvoe 'Calendar View' link.

Please comment on it.

**Comment by TutorReachAdmin** [comment]
We need to show calendar but if this is showing hard to implement please instead show a link to take them to google calendar connections To connect with their calendar

**Comment by samuel7895** [comment]
@TutorReachAdmin 
Is It
Redirect To Profile Settings Calendar Section?

**Comment by AdminTutorreach** [comment]
@Rajar91221  @samuel7895 yes on mobile let them redirect to calender settings to be able to connect their schedule feed to their gmail calender automatically without any complicated technical process
